src/client-config.js

```diff
@@ -27,5 +27,7 @@ window.config = {
   REGISTRATIONS_DASHBOARD_URL: '',
   // http://localhost:4444/public/dashboard/a17e9bc0-15a2-4bd1-92fa-ab0f346227ca#bordered=false&titled=false&refresh=300
   STATISTICS_DASHBOARD_URL: '',
-  FEATURES: {}
+  FEATURES: {},
+  INFORMANT_MINIMUM_AGE: 16,
+  INFORMANT_MAXIMUM_AGE: 80
```

Comment:
@Siyasanga  We do not want these values at all and this PR should instead update all usage of isInformantOfLegalAge in the forms.